### PR TITLE
build wheel with scl python 3.8

### PR DESCRIPTION
**Pull request by jburel**
build wheel with scl python 3.8
Tested via https://github.com/ome/omero-install/pull/260

cc @sbesson

### CODE REVIEW

**Comment by joshmoore** [comment]
Changes seem fine. Will "scl"/"SCL" show up in the wheel name?

**Comment by jburel** [comment]
> Changes seem fine. Will "scl"/"SCL" show up in the wheel name?

No it does not. That should be added to void confusion

**Comment by jburel** [comment]
This is the name convention followed when the wheel is built https://www.python.org/dev/peps/pep-0427/#file-name-convention. In our case `abi tag` is `cp38` so I don't think we can inject scl

**Comment by joshmoore** [comment]
Understood.

**Review by sbesson — CHANGES_REQUESTED** [review]
Can you also update the README so that we clarify the expectations in terms of the Python version and the provenance?

As our general tendency is to move towards SCL Python as the canonical way to get a supported Python 3 version on CentOS 7, I am inclined to get this in and tagged as `0.3.0`. The previous tagged wheels will remain available for Python 3.6 and we can always go for 0.4.0 if we decide to explore an alternative option.

File: Dockerfile [hunk 1]
```diff
@@ -1,6 +1,9 @@
 FROM centos:7
 MAINTAINER ome-devel@lists.openmicroscopy.org.uk
 
+ENV PATH /opt/rh/rh-python38/root/usr/bin:$PATH
+ARG PATH /opt/rh/rh-python38/root/usr/bin:$PATH
```
Comment (sbesson) [inline Dockerfile]: why the duplicate command here?

Comment (jburel) [reply Dockerfile]: thanks for spotting that

**Review by sbesson — APPROVED** [review]

**Pull request merged**